src/TheGuardianIn2020Banner/index.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useState } from 'react';
+import { ThemeProvider } from 'emotion-theming';
+import { Button, buttonReaderRevenueBrandAlt, LinkButton } from '@guardian/src-button';
+import { SvgCross, SvgInfo } from '@guardian/src-icons';
+import { OphanComponentEvent } from '@guardian/types/ophan';
+
+import { BrazeClickHandler } from '../utils/tracking';
+import { styles as commonStyles } from '../styles/bannerCommon';
+import { styles } from './styles';
+
+export type Props = {
+    logButtonClickWithBraze: BrazeClickHandler;
+    submitComponentEvent: (componentEvent: OphanComponentEvent) => void;
+    ophanComponentId?: string;
+    brazeMessageProps: {
+        header?: string;
+        body?: string;
+    };
+};
+
+const catchAndLogErrors = (description: string, fn: () => void): void => {
+    try {
+        fn();
+    } catch (e) {
+        console.log(`Error (${description}): `, e.message);
+    }
+};
+
+export const COMPONENT_NAME = 'TheGuardianIn2020Banner';
+const THE_GU_IN_2020_URL =
+    'https://www.theguardian.com/info/ng-interactive/2019/dec/19/working-report';
+
+export const TheGuardianIn2020Banner: React.FC<Props> = ({
+    logButtonClickWithBraze,
+    submitComponentEvent,
+    ophanComponentId = COMPONENT_NAME,
+    brazeMessageProps: { header, body },
+}: Props) => {
+    const [showBanner, setShowBanner] = useState(true);
+
+    const onCloseClick = (
+        evt: React.MouseEvent<HTMLButtonElement, MouseEvent>,
+        internalButtonId: number,
+    ): void => {
+        evt.preventDefault();
+
+        setShowBanner(false);
+
+        catchAndLogErrors('ophanButtonClick', () => {
```

Comment:
No harm for now, but this little bit here feels like the kind of fiddly and annoying logic we'd want to write once in the future.

Ideally as a function or value we re-use from elsewhere, rather than a part of the component contract. Duplicating most of the rest looks fine to me even beyond this short-term change, means we don't end up cornering ourselves into having all the banners be the same as you say.